src/Bridge/Symfony/Validator/Constraint/Enum.php

```diff
@@ -38,8 +38,6 @@ public function __construct($options)
     {
         parent::__construct($options);
 
-        $this->strict = true;
```

Comment:
This is not the default in Symfony 3, only since sf 4+. I think we should keep it. Or perhaps use `$options['strict'] ?? true` ?

Reply:
You're right we should keep it since it doesn't impact more recent versions of Symfony.